h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostUtilsTest.java

```diff
@@ -688,6 +719,16 @@ private static void comparePreds(final float[][] nativePreds, final Vec h2oPreds
   @RunWith(Parameterized.class)
   public static final class XGBoostSparseMatrixTest extends XGBoostUtilsTest {
 
+    @BeforeClass
+    public static void beforeClass(){
```

Comment:
No need to run both test classes as parametrized, when only one actually is parametrized.